Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Security/Ssl/PortalStats.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace Dnn.PersonaBar.Extensions.Components.Security.Ssl
+{
+    /// <summary>
+    /// Statistics about a portal.
+    /// </summary>
+    public class PortalStats
+    {
+        /// <summary>
+        /// Gets or sets ID of the portal.
+        /// </summary>
+        public int PortalID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total nr of (non-deleted) tabs of the portal.
+        /// </summary>
+        public int NrTabs { get; set; }
```

Comment:
I believe that this field, and the following 2 should be either renamed to "NumberOfTabs" or "TabCount" to provide clear, non-shorted variable names.